source/Calamari/Program.cs

```diff
@@ -74,7 +74,7 @@ protected virtual ContainerBuilder BuildContainer(CommonOptions options)
             builder.RegisterInstance(fileSystem).As<ICalamariFileSystem>();
             builder.RegisterType<VariablesFactory>().AsSelf();
             builder.Register(c => c.Resolve<VariablesFactory>().Create(options)).As<IVariables>().SingleInstance();
-            builder.RegisterType<CombinedScriptEngine>().AsSelf().As<IScriptEngine>();
+            builder.RegisterType<CombinedScriptEngine>().AsSelf().As<ICombinedScriptEngine>();
```

Comment:
Does it need the AsSelf() registration?

Reply:
It did, but doesn't with the latest commit